fuse/file.go

```diff
@@ -3,12 +3,10 @@
 package fuse
 
 import (
+	"context"
 	"errors"
 	"os"
 	"sync"
-	"context"
-	"fmt"
```

Comment:
looks like linter went crazy. We need `fmt` and `log` for this module to compile

Reply:
No, I went crazy. Will fix once I found that bug.